x/ccv/provider/keeper/validator_set_update.go

```diff
@@ -189,3 +190,9 @@ func (k Keeper) FilterValidators(
 
 	return nextValidators
 }
+
+// GetLastBondedValidators iterates the last validator powers in the staking module
+// and returns the first MaxValidators many validators with the largest powers.
+func (k Keeper) GetLastBondedValidators(ctx sdk.Context) []stakingtypes.Validator {
```

Comment:
Why do we need this function? Can't we just call GetLastBondedValidatorsUtil directly?

Reply:
I think the syntax of passing in the staking keeper is unusual, and this way is much, much closer to how one would functions like this expect to look like. It's not necessary, but I think the interface is nicer like this.